ur_robot_driver/include/ur_robot_driver/ros/dashboard_client.h

```diff
@@ -0,0 +1,109 @@
+// this is for emacs file handling -*- mode: c++; indent-tabs-mode: nil -*-
+
+// -- BEGIN LICENSE BLOCK ----------------------------------------------
+// Copyright 2019 FZI Forschungszentrum Informatik
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -- END LICENSE BLOCK ------------------------------------------------
+
+//----------------------------------------------------------------------
+/*!\file
+ *
+ * \author  Felix Mauch mauch@fzi.de
+ * \date    2019-10-21
+ *
+ */
+//----------------------------------------------------------------------
+
+#ifndef UR_ROBOT_DRIVER_ROS_DASHBOARD_CLIENT_H_INCLUDED
+#define UR_ROBOT_DRIVER_ROS_DASHBOARD_CLIENT_H_INCLUDED
+
+#include <ros/ros.h>
+#include <std_srvs/Trigger.h>
+#include <std_srvs/Empty.h>
+
+#include <ur_robot_driver/ur/dashboard_client.h>
+
+namespace ur_driver
+{
+const int DASHBOARD_SERVER_IP = 29999;
```

Comment:
This seems to just be a constant for the port, not an ip.

Reply:
yep, found that, as well :)